src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,60 @@
+const api = 'https://api.spacexdata.com/v3/rockets';
+
+const GET_ROCKETS = 'GET_ROCKETS';
+const RESERVE_ROCKET = 'RESERVE_ROCKET';
+const UNRESERVE_ROCKET = 'UNRESERVE_ROCKET';
+
+// Action Creators
+
+export const getRockets = () => async (dispatch) => {
+  await fetch(api)
+    .then((rockets) => rockets.json())
+    .then(
+      (data) => dispatch({ type: GET_ROCKETS, payload: data }),
+      () => dispatch({ type: GET_ROCKETS, payload: [] }),
```

Comment:
[OPTIONAL] instead of passing the action `{ type: GET_ROCKETS, payload: [] }`, create a function to return that, as best practices, thank you.